grafana-plugin/src/plugin.json

```diff
@@ -3,6 +3,7 @@
   "type": "app",
   "name": "Grafana OnCall",
   "id": "grafana-oncall-app",
+  "preload": true,
```

Comment:
without this Grafana wasn't properly registering our extension points. See the discussion in Slack [here](https://raintank-corp.slack.com/archives/C01C4K8DETW/p1699444399748929?thread_ts=1699444056.495409&cid=C01C4K8DETW) for more context